gen/specs/account_management/spec_formatted_fixed.json

```diff
@@ -4629,127 +4925,949 @@
+                        "in": "path",
+                        "description": "The ID of the required account. \n\n You can find the UUID on the **Account Management** > **Identity & access management** > **OAuth clients** page, during creation of an OAuth client.",
+                        "schema": {
+                            "type": "string"
+                        }
+                    },
+                    {
+                        "name": "subscriptionId",
+                        "required": true,
+                        "in": "path",
+                        "description": "The ID of the required subscription. \n\n Fetch the list of subscriptions with the [GET subscriptions](https://dt-url.net/qd43uld) call.",
+                        "schema": {
+                            "type": "number"
+                        }
+                    },
+                    {
+                        "name": "environmentUuid",
+                        "required": true,
+                        "in": "path",
+                        "description": "The ID of the environment where you want to set the limit. \n\nFetch the list of environments with the [GET environments](https://dt-url.net/6823uuy) request.",
+                        "schema": {
+                            "type": "string"
+                        }
+                    }
+                ],
+                "requestBody": {
+                    "required": true,
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/ClaBudgetLimitDto"
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "200": {
+                        "description": "Success. The response contains the description of the limit.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/ClaBudgetLimitRecordsDto"
+                                }
+                            }
+                        }
+                    }
+                },
+                "tags": [
+                    "Dynatrace Platform Subscription (prior to April 2023)"
+                ],
+                "security": [
+                    {
+                        "bearer": []
+                    }
+                ]
+            }
+        },
+        "/v1/subscriptions/{subscription-uuid}/cost-allocation": {
+            "get": {
+                "operationId": "InsightsController_getCostAllocationBreakdownPage",
+                "summary": "Get usage data by cost allocation field.",
+                "parameters": [
+                    {
+                        "name": "subscription-uuid",
+                        "required": true,
```

Comment:
Compared to `spec_formatted.json` this is `true` instead of `false`.
Any reason for this change?

Reply:
I don't think so, so I will revert it